src/clr-angular/forms/datalist/datalist.spec.ts

```diff
@@ -43,4 +53,13 @@ export default function(): void {
       expect(this.clarityElement.id).toEqual('custom-id');
     });
   });
+
+  describe('Native Datalist Element', function() {
+    spec(ClrDatalist, TestNativeDatalist, undefined, { providers: [] });
```

Comment:
nit: do you need to provide the last two arguments?

Reply:
It doesn't look like it. I'll remove.